src/tsaug/augmentor.py

```diff
@@ -1,15 +1,24 @@
+from typing import List, Tuple, Union, Optional, Callable, Any
+
 import numpy as np
 
 
 class _Augmentor:
-    def __init__(self, augmentor_func=None, is_random=None, **kwargs):
+    def __init__(
+        self,
+        augmentor_func: Optional[Callable[..., np.ndarray]] = None,
```

Comment:
I don't understand what the syntax `Callable[..., np.ndarray]]` means?

Reply:
The first item `...` means any number of arguments and the second is the return value, `np.ndarray`